packages/@ourworldindata/grapher/src/barCharts/DiscreteBarChart.tsx

```diff
@@ -351,9 +352,20 @@ export class DiscreteBarChart
         if (
             this.yColumns.length > 1 &&
             this.selectionArray.numSelectedEntities > 1
-        )
+        ) {
+            // if we have more than one unit, then faceting by entity is not allowed
+            // as comparing multiple units in a single chart isn't meaningful
+            const uniqueUnits = new Set(
+                this.yColumns.map((column) => column.unit)
```

Comment:
I'm not entirely sure about it, but I think we should also use `shortUnit` here (and in AbstractStackedChart).
The reason being similar to what I said elsewhere: [A chart like this](https://ourworldindata.org/grapher/births-and-deaths-projected-to-2100?time=earliest&facet=entity&uniformYAxis=0&country=OWID_WRL~DEU~NAM~BOL) still makes sense, but the long units are "births" and "deaths".

Reply:
Makes perfect sense!